### PR TITLE
Add ESM28q and ESM28q info to MDD-VUMC

### PR DESCRIPTION
**Pull request by mpgxvii**
- Add ESM28q and ESM28q info to MDD-VUMC

*Note: Must also be added to followup branch

### CODE REVIEW

File: RADAR-MDD-VUmc-s1/protocol.json [hunk 1]
```diff
@@ -437,6 +437,162 @@
...
+        "completionWindow": {
+          "unit": "day",
+          "amount": 1
+        },
+        "notification": {
+          "title": {
+            "en": "Dagboek taak begint binnenkort."
+          },
+          "text": {
+            "en": "Klik voor meer information."
```
Comment (yatharthranjan) [inline RADAR-MDD-VUmc-s1/protocol.json:592]: this is not English I presume.

File: RADAR-MDD-VUmc-s1/protocol.json [hunk 2]
```diff
@@ -437,6 +437,162 @@
           "amount": 3
         }
       }
+    },
```
Comment (yatharthranjan) [inline RADAR-MDD-VUmc-s1/protocol.json:440]: Do we need to increment the versions?

**Review by yatharthranjan — COMMENTED** [review]
A few minor comments.

Also, if the followup branch is used in production, we should make it a protected branch so any changes we push must be reviewed in a PR before merging.

**Review by yatharthranjan — APPROVED** [review]

File: RADAR-MDD-VUmc-s1/protocol.json (hunk 1 above)
Comment (mpgxvii) [reply RADAR-MDD-VUmc-s1/protocol.json]: I don't have admin access to make changes to the protected branches. Would it be possible to get access?

**Pull request merged**